libs/comparison_rules.py

```
@@ -0,0 +1,123 @@
+        c (Context): The context object, which contains the source and target dataframes.
+        """
+        source = pd.DataFrame(c.m.source)
+        target = pd.DataFrame(c.m.target)
+
+        if source.equals(target):
+            print(f"Rows match between both source and target")
+            c.s.result = True
+        else:
+            print(f"Rows do not match between both source and target")
+            c.s.result = False
+        c.update
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)